estudios_socioeconomicos/views.py

```diff
@@ -1,3 +1,33 @@
-# from django.shortcuts import render
+from django.views.generic import ListView
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.utils.decorators import method_decorator
 
-# Create your views here.
+
+from perfiles_usuario.utils import is_administrador
+from .models import Estudio
+
+decorators = [user_passes_test(User, is_administrador), login_required]
+
+
+@method_decorator(decorators, name='dispatch')
+class PendientesList(ListView):
```

Comment:
Also why are you using a CBV instead of a regular view? I think we should stick to simple views instead, since at least I don't know how to use these ones. @erikiado @fernandolobato @AndreeToledo please give opinions